test/enrollment_test.rb

```diff
@@ -3,49 +3,40 @@
 require "enrollment"
 
 class EnrollmentTest < Minitest::Test
-  def test_we_can_create_an_instance
-    enrollment = Enrollment.new(name: "ACADEMY 20",
-                                kindergarten_participation: {
+  def setup
+    @enrollment ||= begin
+      enrollment = Enrollment.new(name: "ACADEMY 20",
+                                  kindergarten_participation: {
                                   2010 => 0.3915,
                                   2011 => 0.35356,
-                                  2012 => 0.1234 },
-                               )
-    assert_equal "ACADEMY 20", enrollment.name
-    assert_equal({ 2010 => 0.3915, 2011 => 0.35356, 2012 => 0.1234 },
```

Comment:
Unit tests should follow method naming and functional conventions: single responsibility and in charge of fetching just one piece of data.